build/MainCSSDataURL.m

```diff
@@ -0,0 +1 @@
+NSString* kMainCSSDataURL = @"data:text/css;base64,LyogU21hbGxlciBzaWRlYmFyICovCkBtZWRpYSBhbGwgYW5kIChtaW4td2lkdGg6IDY0MXB4KSB7CiAgICAuXzFlbmggewogICAgICAgIG1pbi13aWR0aDogMjgwcHggIWltcG9ydGFudDsKICAgIH0KfQoKQG1lZGlhIGFsbCBhbmQgKG1heC13aWR0aDogNjQwcHgpIHsKICAgIC8qIFJlbW92ZSBib3JkZXIgZnJvbSBlbnRpcmUgcmlnaHQgcGFuZSwgYWRkIGl0IGp1c3QgdG8gY29udmVyc2F0aW9uICovCiAgICAuXzFxNS0gewogICAgICAgIGJvcmRlci1sZWZ0OiAwICFpbXBvcnRhbnQ7CiAgICB9CiAgICAuXzIwYnAgewogICAgICAgIGJvcmRlci1sZWZ0OiAxcHggc29saWQgcmdiYSgwLCAwLCAwLCAuMjApOwogICAgfQogICAgCiAgICAvKiBNb3ZlIG92ZXIgbmV3IGNvbnZlcnNhdGlvbiBidXR0b24gKi8KICAgIC5fNWwtM1tyb2xlPSJiYW5uZXIiXSB7CiAgICAgICAgZGlzcGxheTpibG9jayAhaW1wb3J0YW50OwogICAgfQogICAgYVtocmVmPSIvbmV3Il0gewogICAgICAgIG1hcmdpbi1yaWdodDotNDlweDsKICAgICAgICBmbG9hdDogcmlnaHQ7CiAgICAgICAgcG9zaXRpb246cmVsYXRpdmU7CiAgICAgICAgei1pbmRleDogMTA7CiAgICB9CiAgICAKICAgIC8qIE1vdmUgb3ZlciBzZWFyY2ggYmFyIHRvIG1ha2Ugcm9vbSBmb3IgYnV0dG9uICovCiAgICAuXzJ5OHkgewogICAgICAgIHBhZGRpbmctbGVmdDogNDFweCAhaW1wb3J0YW50OwogICAgfQogICAgCiAgICAvKiBNYWtlIHN1cmUgdGl0bGUgZG9lc24ndCBvdmVybGFwIHdpdGggYnV0dG9ucyAqLwogICAgLl81NzQzIHsKICAgICAgICBib3gtc2l6aW5nOiBib3JkZXItYm94OwogICAgICAgIHBhZGRpbmctbGVmdDogNDFweCAhaW1wb3J0YW50OwogICAgICAgIHBhZGRpbmctcmlnaHQ6IDQxcHggIWltcG9ydGFudDsKICAgICAgICB3aWR0aDogMTAwJSAhaW1wb3J0YW50OwogICAgICAgIGJveC1zaXppbmc6IGJvcmRlci1ib3g7CiAgICB9CiAgICBoMi5fMTd3MiB7CiAgICAgICAgb3ZlcmZsb3c6IGhpZGRlbjsKICAgICAgICB0ZXh0LW92ZXJmbG93OiBlbGxpcHNpczsKICAgICAgICB3aWR0aDogMTAwJSAhaW1wb3J0YW50OwogICAgfQogICAgCiAgICAvKiBNb3ZlIGdlYXIgaWNvbiB1cCwgcGxhY2UgcmVhZCBpbmRpY2F0b3IgdW5kZXIgZ2VhdCBpY29uICovCiAgICAuXzQtMGgsIC5fMl9hMi5pbWcgewogICAgICAgIGJhY2tncm91bmQtY29sb3I6ICNGRkY7CiAgICAgICAgYm9yZGVyOiAycHggc29saWQgI0ZGRjsKICAgICAgICBib3JkZXItcmFkaXVzOiA1MCU7CiAgICAgICAgbWFyZ2luLXRvcDotNDRweDsKICAgICAgICBtYXJnaW4tcmlnaHQ6LTJweDsKICAgIH0KICAgIAogICAgLyogQ2hhbmdlIGJvcmRlcnMgb24gc2VsZWN0ZWQgY29udmVyc2F0aW9ucyAqLwogICAgLl8xaHQyIC5fNC0waCwgLl8xaHQyIC5fMl9hMi5pbWcgewogICAgICAgIGJvcmRlci1jb2xvcjogI0Y2RjZGNjsKICAgICAgICBiYWNrZ3JvdW5kLWNvbG9yOiAjRjZGNkY2OwogICAgfQp9Cg==";
```

Comment:
Don't include generated files in the source code. There's even a .gitignore rule that should have warned you about this file.